patches/sowm-primitive-multimonitor.patch

```diff
@@ -0,0 +1,95 @@
+diff --git a/Makefile b/Makefile
+index 8573837..72e9542 100644
+--- a/Makefile
++++ b/Makefile
+@@ -1,5 +1,5 @@
+ CFLAGS+= -std=c99 -Wall -Wextra -pedantic
+-LDADD+= -lX11
++LDADD+= -lX11 -lXinerama
+ LDFLAGS=
+ PREFIX?= /usr
+ BINDIR?= $(PREFIX)/bin
+diff --git a/sowm.c b/sowm.c
+index 0cc1293..27c51ce 100644
+--- a/sowm.c
++++ b/sowm.c
+@@ -4,9 +4,10 @@
+ #include <X11/XF86keysym.h>
+ #include <X11/keysym.h>
+ #include <X11/XKBlib.h>
++#include <X11/extensions/Xinerama.h>
+ #include <stdlib.h>
+ #include <signal.h>
+ #include <unistd.h>
+ 
+ typedef union {
+     const char** com;
+@@ -97,6 +99,7 @@ void notify_enter(XEvent *e) {
+ 
+ void notify_motion(XEvent *e) {
+     if (!mouse.subwindow) return;
++    if (cur->f == 1) return;
```

Comment:
This may make sense for me to add to `sowm` itself. Nice change. :)

Reply:
Added this to `sowm`. It can be removed from the patch. I'm also going to add a test for patches to our CI.